HWMgmt/OCPBaselineHardwareManagement.v1_1_0.json

```diff
@@ -0,0 +1,483 @@
+				},
+				"IPv6AddressPolicyTable": {
+					"ReadRequirement": "Recommended"
+				},
+				"IPv6StaticAddresses": {
+					"ReadRequirement": "Recommended"
+				},
+				"IPv6StaticDefaultGateways": {
+					"ReadRequirement": "Recommended"
+				},
+				"IPv6Addresses": {
+					"ReadRequirement": "IfImplemented",
+					"PropertyRequirements": {
+						"Address": {},
+						"PrefixLength": {},
+						"AddressOrigin": {},
+						"AddressState": {}
+					}
+				},
+				"StaticNameServers": {
+					"ReadRequirement": "Recommended"
+				}
+			}
+		},
+		"EthernetInterfaceCollection": {
+			"PropertyRequirements": {
+				"Members": {
+					"MinCount": 1
+				}
+			}
+		},
+		"ManagerAccount": {},
+		"ManagerCollection": {
+			"PropertyRequirements": {
+				"Members": {
+					"MinCount": 1
+				}
+			}
+		},
+		"Manager": {
+			"PropertyRequirements": {
+				"ManagerType": {},
+				"UUID": {},
+				"Status": {},
+				"FirmwareVersion": {},
+				"NetworkProtocol": {},
+				"EthernetInterfaces": {},
+				"LogServices": {},
+				"Links": {
+					"PropertyRequirements": {
+						"ManagerForServers": {
+							"ReadRequirement": "IfImplemented"
+						},
+						"ManagerForChassis": {
+							"ReadRequirement": "IfImplemented"
+						}
+					}
+				}
+			},
+			"ActionRequirements": {
+				"Reset": {
+					"Parameters": {
+						"ResetType": {
+							"MinSupportValues": ["ForceRestart"]
+						}
+					}
+				}
+			}
+		},
+		"ManagerNetworkProtocol": {
+			"PropertyRequirements": {
+				"HostName": {},
+				"Status": {},
+				"FQDN": {},
+				"HTTP": {
+					"PropertyRequirements": {
+						"ProtocolEnable": {},
+						"Port": {}
+					}
+				},
+				"HTTPS": {
+					"PropertyRequirements": {
+						"ProtocolEnable": {},
+						"Port": {}
+					}
+				},
+				"SSH": {
+					"ReadRequirement": "Recommended",
+					"PropertyRequirements": {
+						"ProtocolEnable": {},
+						"Port": {}
+					}
+				},
+				"SSDP": {
+					"ReadRequirement": "Recommended",
+					"PropertyRequirements": {
+						"ProtocolEnable": {},
+						"Port": {},
+						"NotifyMulticstIntervalSeconds": {},
+						"NotifyIPv6Scope": {},
+						"NotifyTTL": {}
+					}
+				}
+			}
+		},
+		"Power": {
+			"PropertyRequirements": {
+				"PowerControl": {
+					"PropertyRequirements": {
+						"PowerConsumedWatts": {
+							"ReadRequirement": "Recommended"
+						},
+						"PowerCapacityWatts": {
+							"ReadRequirement": "Recommended"
+						},
+						"PowerLimit": {
+							"ReadRequirement": "Recommended",
+							"PropertyRequirements": {
+								"LimitInWatts": {},
+								"LimitException": {}
+							}
+						}
+					}
+				}
+			}
+		},
+		"PowerSubsystem": {
+			"PropertyRequirements": {
+				"CapacityWatts": {
+					"Purpose": "Equivalent for Power.PowerControl.PowerCapacityWatts.",
+					"ReadRequirement": "Recommended"
+				}
+			}
+		},
+		"Sensor": {
+			"Purpose": "Equivalent Power.PowerControl.PowerConsumedWatts.",
+			"PropertyRequirements": {
+				"Id": {
```

Comment:
Been thinking about this one a bit... Do we really want to enforce specific values for "Id"? Since clients should be keying off ReadingType and PhysicalContext for programmatic decisions, it might make more sense to put requirements around that to build a "total power" sensor.

Reply:
Reference EnvironmentalMetrics.PowerWatts when subordinate to Chassis. Use usecase.  Control = EnvironentalMetric. PowerLimitWatts. Jeff to propose.